packages/components/src/FigspecViewer/ViewerMixin.ts

```diff
@@ -136,10 +139,10 @@ export const ViewerMixin = <T extends Constructor<LitElement>>(
 
     constructor(...args: any[]) {
       super(...args);
+    }
 
-      this.addEventListener("click", () => {
```

Comment:
Is "Click to unselect" still working?

Reply:
I actually took it out because it was clashing with the inspector panel, so clicking on any element was removing the panel as well, and forgot to put it back! 

Now I did, please check again at https://deploy-preview-12--figspec.netlify.app/